x-pack/heartbeat/monitors/browser/synthexec/synthexec.go

```diff
@@ -244,6 +244,13 @@ func jsonToSynthEvent(bytes []byte, text string) (res *SynthEvent, err error) {
 
 	res = &SynthEvent{}
 	err = json.Unmarshal(bytes, res)
+	stepStatus, _ := res.Payload.GetValue("status")
```

Comment:
I'm wondering if we should make this change in `@elastic/synthetics` so it's in a more strongly typed field than `payload`. @vigneshshanmugam @shahzad31 WDYT?

Reply:
i do agree, moving it to synthetics will make heartbeat code clean

Reply:
++, We should move both the journey and step status out of the payload fields and have a proper schema.

Reply:
Created an issue - https://github.com/elastic/synthetics/issues/230